pkgs/applications/audio/tone/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, fetchNuGet, linkFarmFromDrvs, buildDotnetModule, ffmpeg-full, msbuild, dotnetCorePackages }:
+
+let
+  nugetSource = linkFarmFromDrvs "nuget-packages" (
+    import ./nuget-deps.nix { inherit fetchNuGet; }
+  );
+
+in buildDotnetModule rec {
+  pname = "tone";
+  baseName = pname;
```

Comment:
You are sure baseName is required?

Reply:
Copied it from another dotnet derivation, I don't think so